baseballAPI/ApiModels/BattingStatsCalculator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+
+namespace BaseballAPI.ApiModels
+{
+    public class BattingStatsCalculator : IBattingStatsCalculator
+    {
+        public BattingStats CalculateStats(BattingStats batting)
```

Comment:
This function is modifying the input parameter.
(it is acting as a `decorator` / `populator` of the extra properties)

This is considered a "side effect" in the function.
Not bad, if that is the whole purpose of the function, and it is obvious that is the desired behavior.
But `return`ing the stats from the function makes it hard for client code to realize 
that the thing returned is actually the same instance they passed in.  (and that what they passed in has been changed)

We should either make this function a `void` so it is more obvious that you are modifying the input,
or instead of modifying the input, actually map the values to a new instance that you return,
so the original object is not changed.  (no "side effects")
In many cases returning a new instance is better in how it fits the design,
but in your case it might make more sense to just have this be a `void` function.
(since it is used in the "mapper" class, we know the context for this is in building up a data bag.
your call)